CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Changelog
 
 
-## 0.55.1
+## 0.56.0
```

Comment:
Hum based on the changes we have two different elements here:
  - Remove the `target_column` is not a fix, is a change in the API of the function. We're removing one of the mandatory parameters of the analysis which leads to a major change. If we weren't in the 0.x version, we should release a major because this would break backward compatibility
  - Fix a bug problem on how we retrieve the geometry results from the routing functions. This change alone would be a patch

And thats why I think this is a minor change rather than a patch version